File: agent-connect-android/app/src/main/java/sg/searchhouse/agentconnect/model/api/listing/ListingRemoteOptionPO.kt
package sg.searchhouse.agentconnect.model.api.listing

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class ListingRemoteOptionPO(
    @SerializedName("id")
    val id: Int,
    @SerializedName("videoCallInd")
    var videoCallInd: Boolean
) : Serializable

File: srxproperty/src/dataObject/ListingRemoteOptionPO.js
import {ObjectUtil} from '../utils';

class ListingRemoteOptionPO {
  constructor(data) {
    if (!ObjectUtil.isEmpty(data)) {
      this.id = data.id;
      this.videoCallInd = data.videoCallInd;
    }
  }
}

export {ListingRemoteOptionPO};
